src/lib/create-config.ts

```diff
@@ -431,7 +431,7 @@ export function createConfig(options: IUnleashOptions): IUnleashConfig {
     const frontendApi = options.frontendApi || {
         refreshIntervalInMs: parseEnvVarNumber(
             process.env.FRONTEND_API_REFRESH_INTERVAL_MS,
-            10000,
+            20000,
```

Comment:
updated on purpose. Polling frequency on 20s is still very conservative, and we should be able to go away from polling entirely when we trust the revision id event.